frontend/awx/resources/templates/WorkflowVisualizer/WorkflowVisualizer.tsx

```diff
@@ -103,12 +102,11 @@ export function WorkflowVisualizer() {
       />
     );
   }
-  const [isExpanded] = expanded;
 
   return (
     <VisualizerWrapper isExpanded={isExpanded}>
       <TopologyView
-        isExpanded={isExpanded}
+        $isExpanded={isExpanded}
```

Comment:
PatternFly's TopologyView component doesn't accept the isExpanded prop. Prefixing the prop with a dollar sign ($) will prevent the styled component from passing the prop down to PatternFly's component. 
https://styled-components.com/docs/api#transient-props